app/config/variables.php

```diff
@@ -420,12 +420,93 @@
                 'question' => '',
                 'filter' => ''
             ],
+            [
+                'name' => '_APP_STORAGE_DEVICE',
+                'description' => 'Select default storage device. The default value is \'Local\'. List of supported adapters are \'Local\', \'S3\' and \'DOSpaces\'.',
+                'introduction' => '0.13.0',
+                'default' => 'Local',
+                'required' => false,
+                'question' => '',
+            ],
+            [
+                'name' => '_APP_STORAGE_DEVICE_S3_ACCESS_KEY',
```

Comment:
```suggestion
                'name' => '_APP_STORAGE_S3_ACCESS_KEY',
```
The DEVICE word seems unnecessary as it does not add any additional meaning to the already existing word `_APP_STORAGE_S3_ACCESS_KEY` . I recommend discarding the DEVICE word and keeping the key name shorter.